samples/drivers/adc/boards/nucleo_h743zi.overlay

```diff
@@ -0,0 +1,20 @@
+&adc1 {
+	/* The <channel> doesn't depend on peripheral */
+	/* The <slot> is the request line, so maybe adc1_dma? */
```

Comment:
not sure what that described originally, and what should be described instead

Reply:
Each node in the `dmas` is defined as follows, which is what all the numbers mean.
```
<node-name channel slot channel-config features>
```
The slot is `adc1_dma` I think, as defined in the reference manual in the DMA section, referring to a channel number or something. We need to be able to say what this number means, because it will influence our implementation as well.